.gitlab-ci.yml

```diff
@@ -82,11 +82,16 @@ include:
     file: '.gitlab-ci-github-status-updates.yml'
```

Comment:
Should we then remove these?

Reply:
Yes, you're right, otherwise we get double statuses. I'll remove it.